tests/views_wpunit/Tribe/Events/Views/V2/Day_ViewTest.php

```diff
@@ -0,0 +1,188 @@
+<?php
+
+namespace Tribe\Events\Views\V2;
+
+use Tribe\Events\Views\V2\Views\Day_View;
+use Tribe\Test\PHPUnit\Traits\With_Post_Remapping;
+use Tribe__Events__Main as TEC;
+use DateTime;
+
+/**
+ * Class Month_ViewTest
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Views\V2
+ */
+class Day_ViewTest extends \Codeception\TestCase\WPTestCase {
+	use With_Post_Remapping;
+
+	/**
+	 * When mocking the `date` function this is the value that will be used to generate the date in place of the real
+	 * one.
+	 *
+	 * @var string
+	 */
+	protected $mock_date_value = '2019-03-10 09:00:00';
+
+	/**
+	 * The mock rendering context.
+	 *
+	 * @var \Tribe__Context|\WP_UnitTest_Factory|null
+	 */
+	protected $context;
+
+	public function setUp() {
+		parent::setUp();
+		tribe( 'cache' )->reset();
+
+		tribe_unset_var( \Tribe__Settings_Manager::OPTION_CACHE_VAR_NAME );
+
+		$now = new \DateTime( $this->mock_date_value );
+
+		$this->context = tribe_context()->alter(
+			[
+				'today'      => $this->mock_date_value,
+				'now'        => $this->mock_date_value,
+				'event_date' => $now->format( 'Y-m-d' ),
+			]
+		);
+
+		tribe( 'cache' )->reset();
+	}
+
+	/**
+	 * @test
+	 */
+	public function should_have_next_date() {
+		$timezone_string = 'Europe/Paris';
+		$timezone        = new \DateTimeZone( $timezone_string );
+		update_option( 'timezone_string', $timezone_string );
+
+		$now = new DateTime( $this->mock_date_value, $timezone );
+
+		$events_data = [
```

Comment:
Since you're not running a snapshot test that would require the post IDs to be stable, you do not need the mocking feature at all; just use `tribe_events()` to create events happening at the required dates. This will simplify the setup and make the testing code more robust.

Reply:
That is fair, I was mostly using something i knew was stable for View testing, but it makes total sense to change that.